src/components/atoms/Footer.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 export default function Footer() {
   return (
     <div style={{ textAlignLast: "center" }}>
-      <p>Pixels Forge Studio - Copyright © 2023</p>
+      <p style={{ paddingBottom: "16px" }}>Pixels Forge Studio - Copyright © 2023</p>
```

Comment:
Should we use `Footer.module.scss` here?

Reply:
I ended up not doing this one at the moment because it was very simple but if I improve the footer later yes :)